assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -554,12 +605,19 @@ che.infra.openshift.route.host.domain_suffix=NULL
 
 # Next properties are subject to changes and removal, so do not rely on them in a stable Che assembly
 
-# Docker image of Che plugin broker app that resolves workspace tooling configuration and copies
-# plugins dependencies to a workspace
-#
-# Note these images are overridden by the Che Operator by default; changing the images here will not
+# Docker image of plugin broker metadata.
+# This broker must be run prior to starting the workspaces pod, as its job is to provision required containers,
+# volumes, and environment variables for the workspace to be able to start with the installed plugins enabled.
+# Note this image is overridden by the Che Operator by default; changing the image here will not
 # have an effect if Che is installed via Operator.
 che.workspace.plugin_broker.metadata.image=quay.io/eclipse/che-plugin-metadata-broker:v3.4.0
+
+# Docker image of Che plugin broker artifacts.
+# This broker runs as an init container on the workspace pod. Its job is to take in a list of plugin identifiers
+# (either references to a plugin in the registry or a link to a plugin meta.yaml) and ensure that the correct .vsix
+# and .theia extenions are downloaded into the /plugins directory, for each plugin requested for the workspace.
+# Note this image is overridden by the Che Operator by default; changing the image here will not
```

Comment:
I don't think we need to write here "changing the image here will not have an effect if Che is installed via Operator.". It's implicit for all values set externally with env variables.